igcollect/sfp_dom.py

```diff
@@ -105,54 +152,76 @@ def get_physical_interfaces():
 
 
 def a2_offset_to_celsius(data, offset):
+    return offset_to_celsius(data, SFF_A2_BASE + offset)
+
+
+def a2_offset_to_volts(data, offset):
+    return offset_to_int(data, SFF_A2_BASE + offset) / 10000
+
+
+def a2_offset_to_milliwatts(data, offset):
+    return offset_to_int(data, SFF_A2_BASE + offset) / 10000
+
+
+def a2_offset_to_milliamps(data, offset):
+    return offset_to_int(data, SFF_A2_BASE + offset) / 500
```

Comment:
you could call `offset_to_milliamps()` here, no?

Reply:
Fixed